src/Illuminate/Mail/Transport/MandrillTransport.php

```diff
@@ -45,7 +45,7 @@ public function send(Swift_Mime_Message $message, &$failedRecipients = null)
             'key' => $this->key,
             'to' => $this->getToAddresses($message),
             'raw_message' => $message->toString(),
-            'async' => false,
+            'async' => true,
```

Comment:
I remember it used to be true and I think got changed to `false` in a PR for some reason?